README.md

```diff
@@ -82,11 +82,44 @@ def view_requiring_flag(request):
 def other_view(request):
     return HttpResponse('flag was not set')
 
-@flag_required('MY_FLAG', fallback_view=other_view)
+@flag_required('MY_FALLBACK_FLAG', fallback_view=other_view)
```

Comment:
I don't quite understand this change. To be consistent with the documentation, isn't it that if `MY_FLAG` isn't defined, then `other_view` will be invoked instead of `view_with_fallback`? Or are you just trying to name this flag something else? What about `MY_FLAG_WITH_FALLBACK`?